### ISSUE TITLE
Release of the training code

### ISSUE THREAD

**Issue by Msmy512** [liuxuexun/DBGroup#2]
Hi, great work on the AAAI 2026 paper! I have successfully explored the code for generating pseudo-labels (Stage 1). However, I noticed that the repository currently does not seem to include the code scripts for Stage 2: Self-Training (the 3D network training part using the generated pseudo-labels).Could you kindly release the training scripts?Thank you very much for your time and contribution!
